visualizations/frontend/other/TTable.vue

```diff
@@ -771,17 +772,33 @@ export default {
         column.default_value
       );
       if (cellValue && typeof cellValue === "object") {
-        if (cellValue.rendered) cellValue = cellValue.rendered;
+        if (cellValue.value){
+            cellValue = cellValue.value;
+        } 
         else cellValue = "";
       }
       return cellValue;
     },
     getCellValue(row, column) {
-      const cellValue = this.getRawCellValue(row, column);
+      const cellValue = this.getRenderedCellValue(row, column);
```

Comment:
Only used of `column.format` is false, maybe move inside the ternary operator?

Reply:
Updated